ethereum/contracts/interfaces/relayer/IWormholeRelayer.sol

```diff
@@ -114,6 +299,42 @@ interface IWormholeRelayerSend is IWormholeRelayerBase {
         VaaKey[] memory vaaKeys
     ) external payable;
 
+    /**
+     * @notice Performs a send, and uses the current refund to fund the 'send'
+     * 
+     * Can only be called within execution of 'receiveWormholeMessages', i.e. during a delivery
+     * The refund from the delivery currently in progress will not be sent to the user; it will instead
+     * be paid to the delivery provider to perform the instruction specified here
+     * 
+     * Publishes an instruction for the delivery provider at 'deliveryProviderAddress' 
+     * to relay a payload and VAAs specified by 'vaaKeys' to the address 'targetAddress' on chain 'targetChain' 
+     * with gas limit 'gasLimit' and with msg.value equal to 
+     * receiverValue + (arbitrary amount that is paid for by paymentForExtraReceiverValue of this chain's wei) in targetChain wei.
+     * 
+     * Any refunds will be sent to 'refundAddress' on chain 'refundChain'
+     * targetAddress must implement the IWormholeReceiver interface
```

Comment:
This should probably say that we are now talking about refunds on the new target chain (because current refunds are slapped on to the first forward)